asteroid/engine/system.py

```diff
@@ -52,24 +51,29 @@ def forward(self, *args, **kwargs):
         """
         return self.model(*args, **kwargs)
 
-    def common_step(self, batch, batch_nb):
+    def common_step(self, batch, batch_nb, train=True):
```

Comment:
I guess you cherry-picked three of my commits to have the new `System`.
This is bad habit for two reasons 
- The changes will be asociated with this PR (wrong history)
- If you make a small change on this new version of the file, it is not as obvious to see
There are two ways to deal with this
- Rebase master and force-push
- Merge master into your branch
Because we squash PRs before merging, I prefer the second method.

Reply:
Oh I am sorry man this was a mistake